app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -50,9 +50,6 @@ class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPageChangeL
     val mapListToggle: MapListToggleButton by lazy {
         findViewById(R.id.map_list_toggle)as MapListToggleButton
     }
-    val routeCancelButton: ImageView by lazy {
```

Comment:
Ok that makes sense. However the X is no longer available on the map view either.